src/adapter/etl-adapter-doctrine/src/Flow/ETL/Adapter/Doctrine/DbalQueryExtractor.php

```diff
@@ -42,6 +40,11 @@ public static function single(Connection $connection, string $query, array $para
     public function extract(FlowContext $context) : \Generator
     {
         foreach ($this->parametersSet->all() as $parameters) {
+            /**
+             * @phpstan-ignore-next-line
+             *
+             * @psalm-suppress InvalidArgument
+             */
```

Comment:
This is required due to phpdoc change between 3.x & 4.x, in legacy there is `int` (legacy approach in `ArrayParameterType`).